pkg/splunk/enterprise/util.go

```diff
@@ -32,6 +34,55 @@ import (
 // kubernetes logger used by splunk.enterprise package
 var log = logf.Log.WithName("splunk.enterprise")
 
+// regex to extract the region from the s3 endpoint
+var regionRegex = ".*.s3[-,.](?P<region>.*).amazonaws.com"
+
+//getRegion extracts the region from the endpoint field
+func getRegion(endpoint string) string {
```

Comment:
Can we use the `endpoint` variable directly?

```
func getRegion(endpoint string) string {
	pattern := regexp.MustCompile(regionRegex)
	return pattern.FindStringSubmatch(endpoint)[1]
}
```

Reply:
Yes, we can! I think I left some code as part of the code clean-up. Thanks for catching it :)